docs/understanding-airbyte/operations.md

```diff
@@ -0,0 +1,54 @@
+# Sync Operations
+
+As part of the [connections](connections/README.md) settings, it is possible to add optional operations to a sync.
+
+Here is the list and descriptions of possible (and future) operations:
+
+## dbt transformations
```

Comment:
I think we can better organise this.

What about having two sections:
## Supported Operations
* Dbt transformations (with all the dbt details), as well as a section/link on how to configure the private repo.
## Future Operations
* Webhook
* Airflow
* etc

Reply:
Yes good idea, thanks!